### ISSUE TITLE
Incorrect liquid shows JS output on sync

### ISSUE THREAD

**Issue by DeanmvSG** [mdyd-dev/marketplace-kit#103]
## Expected behavior 
To only display nice formatted error and not JS one

## Actual behavior
Shows nice formatted error AND JS error

## Steps to reproduce the problem
1. Install 2.6.1 (not sure if happened before)
2. Try to sync a file with a liquid error (in this example `{{i,project_cover_image}}` instead of `{{i.project_cover_image}}`
3. Observe error

```
[500] Validation failed: Liquid body syntax is invalid (Liquid syntax error: Expected end_of_string but found comma in "{{i,project_cover_image | json}}")
 {
  "file_path": "form_configurations/community/projects/project_add"
}
Unhandled rejection TypeError: Cannot read property 'refresh_index' of undefined
    at gateway.sync.then.body (/usr/local/lib/node_modules/@platform-os/marketplace-kit/marketplace-kit-watch.js:69:14)
    at tryCatcher (/usr/local/lib/node_modules/@platform-os/marketplace-kit/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/usr/local/lib/node_modules/@platform-os/marketplace-kit/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/usr/local/lib/node_modules/@platform-os/marketplace-kit/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/usr/local/lib/node_modules/@platform-os/marketplace-kit/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/usr/local/lib/node_modules/@platform-os/marketplace-kit/node_modules/bluebird/js/release/promise.js:694:18)
    at _drainQueueStep (/usr/local/lib/node_modules/@platform-os/marketplace-kit/node_modules/bluebird/js/release/async.js:138:12)
    at _drainQueue (/usr/local/lib/node_modules/@platform-os/marketplace-kit/node_modules/bluebird/js/release/async.js:131:9)
    at Async._drainQueues (/usr/local/lib/node_modules/@platform-os/marketplace-kit/node_modules/bluebird/js/release/async.js:147:5)
    at Immediate.Async.drainQueues (/usr/local/lib/node_modules/@platform-os/marketplace-kit/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:810:20)
    at tryOnImmediate (timers.js:768:5)
    at processImmediate [as _immediateCallback] (timers.js:745:5)
```

## Version:

Marketplace Kit: 2.6.1

NodeJs: v8.11.3

Operating System: macOS 10.14.1 (18B75)

**Comment by DeanmvSG** [comment]
Extra info, the above also seems stop the sync from running, saving files afterwards just does "nothing" even after the error is fixed.  Ending and restarting sync works fine 👍

**Comment by pavelloz** [comment]
Got it :)

**Comment by pavelloz** [comment]
Fixed in 2.6.2 :)
